AdminUi/src/AdminUi/Controllers/TiersController.cs

```diff
@@ -43,6 +43,7 @@ public async Task<IActionResult> GetTiersAsync([FromQuery] PaginationFilter pagi
 
     [HttpGet("{tierId}")]
     [ProducesResponseType(typeof(GetTierByIdResponse), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
```

Comment:
Use `[ProducesError(StatusCodes.Status404BadRequest)]` instead. This custom attribute also sets the error payload